it.lproj/Localizable.strings

```diff
@@ -191,10 +191,10 @@
 "Achievements" = "Successi";
 
 /* title for the Day Total awards section of the Achievements area */
-"Day Totals" = "Totali giornalieri";
+"Day Totals" = "Passi giornalieri";
 
 /* title for the Streaks awards section of the Achievements area */
-"Streaks" = "Strisce";
+"Streaks" = "Giorni perfetti";
```

Comment:
In Italian the term "strisce" is not used in this context, furthermore, the objectives of this section refer to rewards given to the user in training as many days as possible. Using "Giorni perfetti" is equivalent to "Perfect days".